### PR TITLE
Feature SWE 1034 uploader logo

### PR DESCRIPTION
**Pull request by Olexandr25**
Wrote Uploader.

Screenshot 2022-07-15 at 11 19 02
Screenshot 2022-07-15 at 11 19 08

https://user-images.githubusercontent.com/36109776/179182943-590fc912-2d64-47e6-bac4-79f461a5dfcb.mp4

### CODE REVIEW

File: src/app/components/Uploader/Uploader.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * It returns a styled component
+ * that contains an uploader wrapper
+ * that contains either an image
+ * or a default picture
+ * @param {string} img - path of image, past in UploaderLog
+ * @param {function} props - all functions for "rc-upload"
+ * @returns A styled component that is a wrapper for the uploader logo.
+ */
+const UploaderLogoRc = (props) => {
```
Comment (olenashevchuk) [inline src/app/components/Uploader/Uploader.js:14]: use destructuring for image

File: src/app/components/Uploader/Uploader.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <UploadStyled {...props}>
+      <UploaderWrapper>
+        {props.image ? (
+          <UploaderLogo type="url" src={props?.image} alt="logo" p={2} />
```
Comment (olenashevchuk) [inline src/app/components/Uploader/Uploader.js:19]: add padding to UploaderLogo component

File: src/app/components/Uploader/Uploader.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        ) : (
+          <DefaultPicture />
+        )}
+      </UploaderWrapper>
+    </UploadStyled>
+  )
+}
+
+export default UploaderLogoRc
```
Comment (olenashevchuk) [inline src/app/components/Uploader/Uploader.js:28]: add prop types

Comment (Olexandr25) [reply src/app/components/Uploader/Uploader.js:28]: Prop types for comments or what?
I am sorry, do not understand you )

File: src/app/components/Uploader/Uploader.js (hunk 1 above)
Comment (olenashevchuk) [reply src/app/components/Uploader/Uploader.js:14]: no

File: src/app/components/Uploader/Uploader.js (hunk 3 above)
Comment (5P4C3GR33D) [reply src/app/components/Uploader/Uploader.js:28]: You use, f.e. `props?.image` right in this component, so minimum this prop should be checked in propTypes.

File: src/app/components/Uploader/Uploader.js [hunk 4]
```diff
@@ -11,12 +12,23 @@ import { ReactComponent as DefaultPicture } from 'app/assets/icons/Shape.svg'
  * @param {function} props - all functions for "rc-upload"
  * @returns A styled component that is a wrapper for the uploader logo.
  */
-const UploaderLogoRc = (props) => {
+const Uploader = (props) => {
+  const { image, ...rest } = props
+  console.log('rest: ', rest)
```
Comment (olenashevchuk) [inline src/app/components/Uploader/Uploader.js:17]: delete console log, please

File: src/app/components/Uploader/Uploader.styled.js [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import { Box, Img } from '@qonsoll/react-design'
 
+// import BorderLogo from ''
```
Comment (olenashevchuk) [inline src/app/components/Uploader/Uploader.styled.js:3]: delete useless import

File: src/app/components/Uploader/Uploader.js (hunk 4 above)
Comment (Olexandr25) [reply src/app/components/Uploader/Uploader.js:17]: in helper function below, delete too?

**Pull request closed without merging**